src/fidesops/service/connectors/query_config.py

```diff
@@ -666,15 +664,47 @@ def generate_query(
         for string_path, reference_values in filtered_data.items():
             for value in reference_values:
                 request_params.append(
-                    self.prepare_params(current_request, {string_path: value})
+                    self.prepare_query_params(current_request, {string_path: value})
                 )
         logger.info(f"Populated request params for {current_request.path}")
         return request_params
 
+    @staticmethod
+    def prepare_update_params(
```

Comment:
It would be useful to see some unit tests on your internals - `prepare_update_params` and `prepare_query_params`, you have a nice end-to-end test but I wonder if we're missing corner cases

Reply:
I have some unit tests under the `TestSaaSQueryConfig` class and I can add some more as we discover new use cases in the next body of work.